docker_sample/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rust:1.41.0-stretch as builder
+
+# muslc is required in order to build the rust image.
+RUN apt-get update && apt-get -y install ca-certificates cmake musl-tools libssl-dev && rm -rf /var/lib/apt/lists/*
+
+COPY . .
+
+# Sets the environment variable for the cargo build command that follows.
+ENV PKG_CONFIG_ALLOW_CROSS=1
+
+RUN cargo build --release
+
+FROM rust:1.41.0-stretch
+
+#RUN apk --no-cache add ca-certificates 
```

Comment:
If this command is not required I think it should be left here on the Dockerfile.